[2023-10-27 14:17:14,947] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 14:17:14,955] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 14:17:14,955] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:17:14,955] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:17:14,956] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:17:14,971] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-11 00:00:00+00:00
[2023-10-27 14:17:14,975] {standard_task_runner.py:52} INFO - Started process 17509 to run task
[2023-10-27 14:17:14,987] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-11T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpuc9ot35w', '--error-file', '/tmp/tmppcb3plhe']
[2023-10-27 14:17:14,997] {standard_task_runner.py:80} INFO - Job 31: Subtask GOOG
[2023-10-27 14:17:15,063] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:17:15,118] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-11T00:00:00+00:00
[2023-10-27 14:17:15,598] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/gabriel/Documents/project_finance/airflow/dags/get_finance.py", line 18, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/core/generic.py", line 3902, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/format.py", line 1152, in to_csv
    csv_formatter.save()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/csvs.py", line 247, in save
    with get_handle(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 739, in get_handle
    check_parent_directory(str(handle))
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 604, in check_parent_directory
    raise OSError(rf"Cannot save file into a non-existent directory: '{parent}'")
OSError: Cannot save file into a non-existent directory: '/home/gabriel/Documents/project_finance/datalake/stocks/GOOG'
[2023-10-27 14:17:15,660] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231011T000000, start_date=20231027T171714, end_date=20231027T171715
[2023-10-27 14:17:15,696] {standard_task_runner.py:92} ERROR - Failed to execute job 31 for task GOOG (Cannot save file into a non-existent directory: '/home/gabriel/Documents/project_finance/datalake/stocks/GOOG'; 17509)
[2023-10-27 14:17:15,737] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-10-27 14:17:15,754] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 14:30:49,898] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 14:30:49,919] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 14:30:49,919] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:30:49,919] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:30:49,919] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:30:49,939] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-11 00:00:00+00:00
[2023-10-27 14:30:49,945] {standard_task_runner.py:52} INFO - Started process 19830 to run task
[2023-10-27 14:30:49,952] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-11T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmp1dawoksz', '--error-file', '/tmp/tmp4zzc5fg3']
[2023-10-27 14:30:49,953] {standard_task_runner.py:80} INFO - Job 31: Subtask GOOG
[2023-10-27 14:30:50,028] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:30:50,114] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-11T00:00:00+00:00
[2023-10-27 14:30:50,712] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/gabriel/Documents/project_finance/airflow/dags/get_finance.py", line 20, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/core/generic.py", line 3902, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/format.py", line 1152, in to_csv
    csv_formatter.save()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/csvs.py", line 247, in save
    with get_handle(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 863, in get_handle
    handle = open(
PermissionError: [Errno 13] Permission denied: '/GOOG_20231011.csv'
[2023-10-27 14:30:50,779] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231011T000000, start_date=20231027T173049, end_date=20231027T173050
[2023-10-27 14:30:50,872] {standard_task_runner.py:92} ERROR - Failed to execute job 31 for task GOOG ([Errno 13] Permission denied: '/GOOG_20231011.csv'; 19830)
[2023-10-27 14:30:50,910] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-10-27 14:30:50,933] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 14:34:41,140] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 14:34:41,181] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 14:34:41,181] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:34:41,181] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:34:41,182] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:34:41,218] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-11 00:00:00+00:00
[2023-10-27 14:34:41,274] {standard_task_runner.py:52} INFO - Started process 20796 to run task
[2023-10-27 14:34:41,290] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-11T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmp4a1ns2bi', '--error-file', '/tmp/tmplafyphtn']
[2023-10-27 14:34:41,291] {standard_task_runner.py:80} INFO - Job 31: Subtask GOOG
[2023-10-27 14:34:41,373] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:34:41,563] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-11T00:00:00+00:00
[2023-10-27 14:34:42,186] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 14:34:42,197] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231011T000000, start_date=20231027T173441, end_date=20231027T173442
[2023-10-27 14:34:42,232] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 14:34:42,274] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 14:54:41,922] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 14:54:41,929] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 14:54:41,929] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:54:41,929] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 14:54:41,929] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 14:54:41,939] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-11 00:00:00+00:00
[2023-10-27 14:54:41,943] {standard_task_runner.py:52} INFO - Started process 23489 to run task
[2023-10-27 14:54:41,947] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-11T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpd6c4m3_h', '--error-file', '/tmp/tmp_ve45h9b']
[2023-10-27 14:54:41,947] {standard_task_runner.py:80} INFO - Job 31: Subtask GOOG
[2023-10-27 14:54:41,988] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 14:54:42,032] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-11T00:00:00+00:00
[2023-10-27 14:54:42,500] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 14:54:42,505] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231011T000000, start_date=20231027T175441, end_date=20231027T175442
[2023-10-27 14:54:42,552] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 14:54:42,622] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 15:23:53,733] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 15:23:53,742] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 15:23:53,742] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 15:23:53,742] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 15:23:53,742] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 15:23:53,753] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-11 00:00:00+00:00
[2023-10-27 15:23:53,757] {standard_task_runner.py:52} INFO - Started process 27338 to run task
[2023-10-27 15:23:53,760] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-11T00:00:00+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmpn2088bcz', '--error-file', '/tmp/tmpzhjsa4vp']
[2023-10-27 15:23:53,761] {standard_task_runner.py:80} INFO - Job 31: Subtask GOOG
[2023-10-27 15:23:53,810] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 15:23:53,869] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-11T00:00:00+00:00
[2023-10-27 15:23:54,574] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 15:23:54,581] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231011T000000, start_date=20231027T182353, end_date=20231027T182354
[2023-10-27 15:23:54,637] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 15:23:54,667] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 17:02:23,161] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 17:02:23,175] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 17:02:23,175] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:02:23,176] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 17:02:23,176] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:02:23,192] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-11 00:00:00+00:00
[2023-10-27 17:02:23,198] {standard_task_runner.py:52} INFO - Started process 39567 to run task
[2023-10-27 17:02:23,201] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-11T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmp16ghj1sk', '--error-file', '/tmp/tmpdaxjl46z']
[2023-10-27 17:02:23,202] {standard_task_runner.py:80} INFO - Job 37: Subtask GOOG
[2023-10-27 17:02:23,238] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 17:02:23,280] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-11T00:00:00+00:00
[2023-10-27 17:02:24,277] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/gabriel/Documents/project_finance/airflow/dags/get_finance.py", line 20, in get_history
    yfinance.Ticker(ticker).history(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/core/generic.py", line 3902, in to_csv
    return DataFrameRenderer(formatter).to_csv(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/format.py", line 1152, in to_csv
    csv_formatter.save()
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/formats/csvs.py", line 247, in save
    with get_handle(
  File "/home/gabriel/Documents/project_finance/venv/lib/python3.9/site-packages/pandas/io/common.py", line 863, in get_handle
    handle = open(
IsADirectoryError: [Errno 21] Is a directory: '/home/gabriel/Documents/project_finance/datalake/Bronze/stocks/GOOG'
[2023-10-27 17:02:24,284] {taskinstance.py:1395} INFO - Marking task as FAILED. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231011T000000, start_date=20231027T200223, end_date=20231027T200224
[2023-10-27 17:02:24,292] {standard_task_runner.py:92} ERROR - Failed to execute job 37 for task GOOG ([Errno 21] Is a directory: '/home/gabriel/Documents/project_finance/datalake/Bronze/stocks/GOOG'; 39567)
[2023-10-27 17:02:24,312] {local_task_job.py:156} INFO - Task exited with return code 1
[2023-10-27 17:02:24,328] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 17:09:41,147] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 17:09:41,154] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 17:09:41,154] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:09:41,154] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 17:09:41,154] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 17:09:41,165] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-11 00:00:00+00:00
[2023-10-27 17:09:41,170] {standard_task_runner.py:52} INFO - Started process 41287 to run task
[2023-10-27 17:09:41,173] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-11T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmp66l_o307', '--error-file', '/tmp/tmp8mcxeuoz']
[2023-10-27 17:09:41,174] {standard_task_runner.py:80} INFO - Job 37: Subtask GOOG
[2023-10-27 17:09:41,215] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 17:09:41,268] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-11T00:00:00+00:00
[2023-10-27 17:09:41,778] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 17:09:41,783] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231011T000000, start_date=20231027T200941, end_date=20231027T200941
[2023-10-27 17:09:41,842] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 17:09:41,859] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-27 18:10:33,744] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 18:10:33,752] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [queued]>
[2023-10-27 18:10:33,752] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 18:10:33,752] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2023-10-27 18:10:33,752] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-10-27 18:10:33,763] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): GOOG> on 2023-10-11 00:00:00+00:00
[2023-10-27 18:10:33,767] {standard_task_runner.py:52} INFO - Started process 49442 to run task
[2023-10-27 18:10:33,771] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_stocks_dag', 'GOOG', 'scheduled__2023-10-11T00:00:00+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/get_finance.py', '--cfg-path', '/tmp/tmp5fbi_657', '--error-file', '/tmp/tmpxcmvt_cz']
[2023-10-27 18:10:33,771] {standard_task_runner.py:80} INFO - Job 36: Subtask GOOG
[2023-10-27 18:10:33,814] {task_command.py:370} INFO - Running <TaskInstance: get_stocks_dag.GOOG scheduled__2023-10-11T00:00:00+00:00 [running]> on host airflow-virtual-machine
[2023-10-27 18:10:33,865] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_stocks_dag
AIRFLOW_CTX_TASK_ID=GOOG
AIRFLOW_CTX_EXECUTION_DATE=2023-10-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-11T00:00:00+00:00
[2023-10-27 18:10:34,353] {python.py:173} INFO - Done. Returned value was: None
[2023-10-27 18:10:34,358] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_stocks_dag, task_id=GOOG, execution_date=20231011T000000, start_date=20231027T211033, end_date=20231027T211034
[2023-10-27 18:10:34,510] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-27 18:10:34,598] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
